CT-1806 Hide SARs reports behind feature flag

stephenrichards opened the pull request:
SARs reports were originally hidden behind the sars feature flag.

Becuase we want to release SARs functionality before the SARs reporting,
we now hide the reports behind a sars_reporting flag.

misaka commented:
The failed test is the sporadic error we get in the upload feature test. Not related to my changes. Merging.

stephenrichards committed:
CT-1806 Hide SARs reports behind feature flag

SARs reports were originally hidden behind the sars feature flag.

Becuase we want to release SARs functionality before the SARs reporting,
we now hide the reports behind a sars_reporting flag.

beccapearce committed:
Merge branch 'master' into ct-1805-sars-reporting-feature-flag

stephenrichards committed:
Merge branch 'master' into ct-1805-sars-reporting-feature-flag
